frontend/app/src/people/widgetViews/summaries/wantedSummaries/CodingMobile.tsx

```diff
@@ -161,8 +161,7 @@ export default function MobileView(props: CodingViewProps) {
             />
           </ButtonRow>
 
-          {!!paid ? markUnpaid : null}
-
+          {!!paid ? markUnpaid : markUnpaid}
```

Comment:
this doesnt really make sense we should just change the prop name to `markPaidOrUnpaid` and then just display it at all times

Reply:
can we remove this line as its not needed ?

Reply:
Done